README.md

```diff
@@ -0,0 +1,36 @@
+# Percona Distribution for PostgreSQL documentation.
+
+Welcome to Percona Distribution for PostgreSQL documentation!
+
+## Overview
+
+Percona Distribution for PostgreSQL is a collection of tools to assist you in managing your PostgreSQL database system. It includes the upstream version of PostgreSQL and a selection of extensions that enable solving essential practical tasks efficiently.
+ 
+This repository contains the source files for [Percona Distribution for PostgreSQL documentation](https://www.percona.com/doc/postgresql/11/index.html). The documentation is written in [reStructured text markup language](https://docutils.sourceforge.io/rst.html) and is created using [Sphinx Python Documentation Generator](https://www.sphinx-doc.org/en/master/). 
+
+## Installation
```

Comment:
I think this section should focus on this documentation, not so much about the actual distribution. So instead of installing, how about adding instructions on how to **build** the documentation locally?

Reply:
I have doc building instructions in the Contributing guide: https://github.com/percona/postgresql-docs/pull/57

Reply:
In that case the `Installation` and `Submitting bug reports` could likely be removed, as the former one does not apply to the documentation and the latter is not applicable and somewhat redundant to what's written in the proposed `CONTRIBUTING.md`?

Consider reading this document as somebody who's interested in contributing to the content of **this repository** (the PG distribution documentation). Instructions on how to install or report bugs in the PG distribution should be included in the actual documentation instead.